FRAuth/FRAuthTests/FRAuthSwiftTests/FRAuth/Model/OAuth2/TokenTests.swift

```diff
@@ -63,6 +75,39 @@ class TokenTests: FRAuthBaseTest {
         }
     }
     
+    func testTokenSecureCodingWithoutDefaultValues() {
+        if #available(iOS 11.0, *) {
```

Comment:
Why 11? We are on 12 going to 13 now right? We might be able to up this a bit if not mistaken

Reply:
This has been addressed...